ivy_tests/test_ivy/test_frontends/test_jax/test_numpy/test_statistical.py

```diff
@@ -65,6 +66,42 @@ def test_jax_einsum(
     )
 
 
+# histogram_bin_edges
+@handle_frontend_test(
+    fn_tree="jax.numpy.histogram_bin_edges",
+    values=histogram_helper(),
```

Comment:
Most of the things returned from the `histogram_helper` could be obtained using existing helper functions, for example `a`, `bins`, `weights` etc are normal arrays which could be easily obtained using existing helpers. I would recommend going through the [testing deep dive](https://unify.ai/docs/ivy/overview/deep_dive/ivy_tests.html) or the [hypothesis helpers](https://github.com/unifyai/ivy/tree/main/ivy_tests/test_ivy/helpers/hypothesis_helpers) to learn about the functions that suit your use-case.